### DISCUSSION TITLE
What keyboard is this?

### DISCUSSION THREAD

**Discussion by augustobob** [samolego/Canta#46] [General]
https://github.com/samolego/Canta/assets/34912839/ef7c91fe-d821-42a4-b699-0e83be591ff6

**Comment by samolego** [comment]
Florisboard, I highly recommend checking it out :)

https://github.com/florisboard/florisboard/

**Comment by augustobob** [comment]
Thank you, unfortunately my android device is 4.1 but Ill keep track of the keyboard when I update my phone
